src/utils.js

```diff
@@ -535,6 +535,30 @@ function serializeKeysForMessage(keys, maxLength) {
   return '';
 }
 
+function sanitize(input, sanitizeKeys) {
+  sanitizeKeys = isArray(sanitizeKeys) ? sanitizeKeys : [];
+  var sanitizeMask = '********';
```

Comment:
i know we use asterisks elsewhere, but i think something explicit like `[value sanitized by raven.js]`could be better.

probably not worth breaking with the precedent though